src/widgets/VaccineChart.js

```diff
@@ -155,40 +265,77 @@ export class VaccineChart extends React.Component {
   }
 }
 
+const chartSize = {
+  width: 1,
+  height: 1.025,
+};
+
 const chartStyles = {
   display: 'flex',
   flexDirection: 'row',
   justifyContent: 'center',
 };
 
+const axisStyles = {
+  fontSize: 15,
+  fontFamily: APP_FONT_FAMILY,
+  fill: '#909192',
+};
+
+const minBoundaryStyles = {
+  stroke: '#70b4f0',
```

Comment:
should probably add this color to 'colours'. Maybe call it MIN_TEMPERATURE_BLUE

Reply:
Fixed bty 4a915d3. I went for `WARM_BLUE`, as seems a bit more generic, but can change.